packages/blockchain-wallet-v4-frontend/src/data/modules/coinify/sagas.js

```diff
@@ -158,8 +163,13 @@ export default ({ coreSagas }) => {
       const modals = yield select(selectors.modals.getModals)
       const trade = yield select(selectors.core.data.coinify.getTrade)
 
-      if (path(['type'], head(modals)) === 'CoinifyExchangeData') yield put(actions.modals.closeAllModals())
-      else yield put(actions.form.change('buySellTabStatus', 'status', 'order_history'))
+      if (path(['type'], head(modals)) === 'CoinifyExchangeData') {
+        yield put(A.coinifySignupComplete())
+        yield call(delay, 500)
```

Comment:
Why delay?

Reply:
so that when the user is done with KYC, the step indicator can briefly go to "100%" before the modal closes

Reply:
Nice!